apps/concierge_site/lib/schedule.ex

```diff
@@ -240,6 +237,15 @@ defmodule ConciergeSite.Schedule do
       } = departure_schedule
 
       %{"attributes" => %{"arrival_time" => arrival_timestamp}} = arrival_schedule
+
+      if is_nil(departure_timestamp) do
+        raise "departure_timestamp is nil"
+      end
```

Comment:
I don't think these explicit raises are needed. The error that will be raised when we try to convert the time string to a time is very clear already:

```
** (FunctionClauseError) no function clause matching in Calendar.ISO.parse_naive_datetime/1

    The following arguments were given to Calendar.ISO.parse_naive_datetime/1:

        # 1
        nil

    Attempted function clauses (showing 1 out of 1):

        def parse_naive_datetime(string) when is_binary(string)
```